src/lib.rs

```diff
@@ -94,6 +98,34 @@ pub fn run() -> Result<()> {
             &image,
             img_to_bmap_path!(generate_bmap, &image),
         )?,
+        Command::Ssh(SshConfig {
+            device,
+            username,
+            dir,
+            priv_key_path,
+            config_path,
+        }) => {
+            #[tokio::main]
+            async fn create_ssh_tunnel(
+                device: &str,
+                username: &str,
+                dir: Option<PathBuf>,
+                priv_key_path: Option<PathBuf>,
+                config_path: Option<PathBuf>,
+            ) -> Result<()> {
+                let access_token = crate::auth::authorize(&*crate::auth::AUTH_INFO_DEV)
+                    .await
+                    .context("create ssh tunnel")?;
+
+                let config = ssh::Config::new(dir, priv_key_path, config_path)?;
+
+                ssh::ssh_create_tunnel(device, username, config, access_token).await?;
+
+                Ok(())
```

Comment:
can be shortened by:?
`ssh::ssh_create_tunnel(device, username, config, access_token).await`